racket/collects/syntax/parse/experimental/template.rkt

```diff
@@ -4,7 +4,8 @@
                      racket/syntax
                      syntax/parse/private/minimatch
                      racket/private/stx ;; syntax/stx
-                     racket/private/sc)
+                     racket/private/sc
+                     racket/struct)
```

Comment:
Avoid depending on `racket/struct`, since it pulls in `racket/contract/base` (and see below)